postgres/rdspostgres/rdspostgres_test.go

```diff
@@ -104,6 +61,7 @@ func TestOpenWithURL(t *testing.T) {
 		{fmt.Sprintf("rdspostgres://%s:%s@%s/wrongdbname", username, password, endpoint), false, true},
 		{fmt.Sprintf("rdspostgres://%s:%s@%s/%s?foo=bar", username, password, endpoint, databaseName), false, true},
```

Comment:
It looks like you deleted all of the tests verifying that `sslmode` etc. are rejected, can you add some back here?

Reply:
They are already covered by other test cases, see ~10 lines above.

Reply:
Oh sorry, GitHub being clever and hiding lines.